frontend/src/components/UserModal/index.js

```diff
@@ -89,16 +151,22 @@ const UserModal = ({ open, onClose, userId }) => {
 	const [selectedQueueIds, setSelectedQueueIds] = useState([]);
 	const [showPassword, setShowPassword] = useState(false);
 	const [whatsappId, setWhatsappId] = useState(false);
-	const {loading, whatsApps} = useWhatsApps();
+	const { loading, whatsApps } = useWhatsApps();
+	const { handleProfileImage, handleUser } = useContext(ProfileImageContext);
 
 	useEffect(() => {
 		const fetchUser = async () => {
 			if (!userId) return;
 			try {
 				const { data } = await api.get(`/users/${userId}`);
+				const { profileImage } = data;
+				const profileUrl = profileImage ? `http://localhost:8080/profilePics/${profileImage}` : null;
```

Comment:
Uma atenção neste ponto.
Desta forma que voce escreveu o link da imagem está estática, na hora que nao for mais localhost acredito que sua lógica nao funcionará.
Verifica este ponto, e se realmente sua logica nao funcionar, faz o seguinte: No arquivo .env no frontend temos o link para a url do backend, ai so chama a funcao getBackendUrl() que esta no arquivo config.jsx e complementa com o /profilePics/${profileImage}

Reply:
Obrigado pela sujestão! Já inseri as mudanças no próximo commit.